src/test/java/net/masterthought/cucumber/ConfigurationTest.java

```diff
@@ -56,36 +55,66 @@ public void isParallelTesting_ReturnsParallelTesting() {
     }
 
     @Test
-    public void getJenkinsBasePath_OnSampleBath_ReturnsJenkinsPath() {
+    public void getJenkinsBuildURL_OnSampleURL_ReturnsJenkinsBuildURL() {
 
         // give
         Configuration configuration = new Configuration(outputDirectory, projectName);
-        String basePath = "abc321";
-        configuration.setJenkinsBasePath(basePath);
+        String jobPath = "http://localhost:8080/jobs/TEST/2/";
```

Comment:
right, in tests when you are comparing to this value in assertion it is nice to make it ummutable